apps/test/integration/levelSolutions/applab/ec_data_blocks.js

```diff
@@ -92,10 +92,7 @@ export default {
       customValidator(assert) {
         // Verify that onRecordEvent was called with the correct data
         const debugOutput = document.getElementById('debug-output');
-        assert.equal(
-          debugOutput.textContent,
-          'create 2\n' + 'update 1\n' + 'delete 1'
-        );
+        assert.equal(debugOutput.textContent, '"create 2""update 1""delete 1"');
```

Comment:
@kimj42 and @ajpal - It looks like we format these strings in two different ways: `"create 2""update 1""delete 1"` and `"create 1"' + '"create 2"' + '"update 1"' + '"delete 1"`. I'm fine with either but I think we should be consistent. Thoughts?

Reply:
On second thought, it looks like we were using the `+` version so I'm going to go with that.